praetorian_cli/handlers/add.py

```
@@ -5,11 +5,29 @@
+        if cmd is not None:
+            return cmd
+
+        # Return a dynamic command that adds entities with the label=cmd_name
+        @click.command(name=cmd_name)
+        @click.option('-e', '--entries', required=True, help='key value pairs that make up the object')
+        @cli_handler
+        def _dynamic(chariot, entries):
+            try:
+                entries_data = get_dict_from_entries(entries)
+                print_json(chariot.generic.add(cmd_name, entries_data))
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)